harbor/README-v2.13.1.md

```diff
@@ -0,0 +1,18 @@
+| persistence.persistentVolumeClaim.database.storageClass	         | A storage policy name.             | 	A storage class that is used for the Harbor database PVCs.                                                                                                                                                                                                                                                                                   |
+| persistence.persistentVolumeClaim.redis.storageClass	            | A storage policy name.             | 	A storage class that is used for the Harbor redis PVCs.                                                                                                                                                                                                                                                                                      |
+| persistence.persistentVolumeClaim.trivy.storageClass	            | A storage policy name.             | 	A storage class that is used for Harbor trivy PVCs.                                                                                                                                                                                                                                                                                          |
+| enableNginxLoadBalancer 	                                        | true or false                      | Use a K8s Service of type LoadBalancer to expose Harbor's endpoints when it's set to true.  This requires a Supervisor to be configured with a load balancer.  enableNginxLoadBalancer and enableContourHttpProxy can't be true at the same time.  When they are both set to false, an Ingress will be created to expose Harbor's endpoints.	 |
+| enableContourHttpProxy	                                          | true or false                      | When true, uses Contour's httpproxy resources to expose Harbor's endpoint. Ensure enableNginxLoadBalancer and enableContourHttpProxy are not both set to true.	                                                                                                                                                                               | 
+| createNetworkPolicy 	                                            | true or false                      | Make sure this is set to true, otherwise there will be issue when Harbor is exposed via Contour in NSXT network	                                                                                                                                                                                                                              |
```

Comment:
Does it need to be set to true when using NSX with enableNginxLoadBalancer?
This description is not very descriptive, can you elaborate?

Reply:
Harbor will be accessible without networkpolicy with "enableNginxLoadBalancer" in NSXT network.